packages/docusaurus/package.json

```diff
@@ -79,6 +79,7 @@
     "html-minifier-terser": "^5.1.1",
     "html-tags": "^3.1.0",
     "html-webpack-plugin": "^5.3.1",
+    "http-proxy": "^1.18.1",
```

Comment:
not too fan of introducing another static asset serve library for a specific case

those 2 libs may behave differently regarding a few things like pretty URLs, trailing slashes etc so I'd rather keep only one